### DISCUSSION TITLE
任务栏崩溃

### DISCUSSION THREAD

**Discussion by PingfanJ** [zhongyang219/MusicPlayer2#653] [Q&A]
在Windows11下有时候用着用着任务栏就会卡住不动，不知道是不是因为和TranslucentTB（一个任务栏美化软件）一起用的关系
以前没用TranslucentTB的时候似乎没发生过任务栏卡住，也可能是我没注意到过

**Comment by lrisora** [comment]
试试这个
https://github.com/zhongyang219/MusicPlayer2/actions/runs/7288242360

**Reply by PingfanJ** [reply]
似乎是解决了，感谢！
